### PR TITLE
Add browserify-css

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: test/karma-cov.conf.js [hunk 1]
```diff
@@ -4,7 +4,9 @@ module.exports = function (config) {
     frameworks: ['browserify', 'mocha'],
     reporters: ['mocha', 'coverage'],
     preprocessors: { 'build/index.js': ['browserify'] },
-    browserify: { debug: true, transform: ['browserify-istanbul'] },
+    browserify: { debug: true,
```
Comment (sccolbert) [inline test/karma-cov.conf.js]: 1 line is fine here

File: src/index.ts [hunk 2]
```diff
@@ -7,6 +7,8 @@
 |----------------------------------------------------------------------------*/
 'use strict';
 
+import './index.css';
```
Comment (sccolbert) [inline src/index.ts]: let's put this import last

**Comment by sccolbert** [comment]
Can you add a note to the README about bundling for browser use?

**Comment by blink1073** [comment]
How's that?

File: package.json [hunk 3]
```diff
@@ -7,6 +7,8 @@
     "definition": "lib/phosphor-widget.d.ts"
   },
   "dependencies": {
+    "browserify": "^11.0.1",
+    "browserify-css": "^0.6.1",
```
Comment (sccolbert) [inline package.json]: These shouldn't be runtime dependencies

Comment (blink1073) [inline package.json]: Fixed.

File: README.md [hunk 4]
```diff
@@ -80,6 +80,19 @@ Earlier versions may also work, but come with no guarantees.
...
 
+Bundling for Browser Use
+------------------------
+
+Follow the package install instructions first.
+
+Bundling requires `browserify` and `browserify-css`.
+
+```bash
+npm install --save browserify browserify-css
```
Comment (sccolbert) [inline README.md]: s/--save/--save-dev

**Pull request merged**